File: exercise15.rb
def splitToTwoChars string
  array_of_strings = []
  (1..string.length).step(2) do |i|
    array_of_strings.push([]) if i.odd?
    array_of_strings[-1].push(string[i - 1] + string[i])
  end
  array_of_strings
end

string = "abcdef"
p splitToTwoChars(string)

File: mini-exercise/exercise15.js
function splitToTwoChars(string) {
  let arrayOfStrings = [];
  for (let i=1; i < string.length; i+= 2) {
    arrayOfStrings.push([]);
    arrayOfStrings[arrayOfStrings.length - 1].push(string[i-1] + string[i])
  }
  return arrayOfStrings
}

console.log(splitToTwoChars("abcdef"))
